build.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+rm -rf CMakeFiles
+rm CMakeCache.txt
+cmake .
+make 
```

Comment:
Normally when you build you dont want to clean your cache, so this script should probably not do that